deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
+LimitNOFILE='8192'
+
+Environment=JENKINS_HOME=/var/lib/jenkins
+ExecStart=/usr/bin/java \
+    -jar /usr/share/jenkins/jenkins.war \
+    --webroot=/var/cache/jenkins/war \
+    --pluginroot=/var/cache/jenkins/plugins \
+    --httpPort='8080' \
+
+# We send SIGTERM(15), java exit code is 143 = 128 + 15
+SuccessExitStatus=143
```

Comment:
really - jenkins sucessfull exit should be zero - seems like something is wrong here.

Most likely this should not be using `SIG_TERM` but `SIG_INT`.